packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -59,12 +59,17 @@ export class BlockList extends Component {
 	}
 
 	renderDefaultBlockAppender() {
+		const { shouldShowInsertionPointBefore } = this.props;
```

Comment:
Just curious if it was intentional to extract `shouldShowInsertionPointBefore` from `this.props`, but not to also extract `this.props.rootClientId` and `this.props.renderAppender`? I'm only asking in case there's a good reason to do one but not the other that I don't know about.

Reply:
No particular reason.

Extracted `shouldShowInsertionPointBefore` to [mirror code elsewhere in the module](https://github.com/WordPress/gutenberg/blob/2e5589310371cdb6ee557df1380fa56fa06a4fc3/packages/block-editor/src/components/block-list/index.native.js#L125), while the `this.props.rootClientId` and `this.props.renderAppender` is code I didn't need to touch other than indent right.

Is there a particular logic we follow for when to extract props? I'd like to revise the code to comply if so 👍.

Reply:
>Is there a particular logic we follow for when to extract props?

Not that I know of.

Reply:
Cool, let leave it as is then 👍